app/src/main/res/values/strings.xml

```diff
@@ -36,4 +36,5 @@
     <string name="firebase_user_management">Firebase User Management</string>
     <string name="auth_failed">Authentication failed</string>
     <string name="action_bar_settings_label">Settings</string>
+    <string name="title_activity_maps">Map</string>
```

Comment:
When naming these variables, you should try to move from most generic to most specific, so something like `activity_maps_label` would be better.